websites/A/Anime Notifier/dist/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "author": {
+    "name": "BasicallyWiz",
+    "id": "324588568951390220"
+  },
+  "service": "Anime Notifier",
+  "description": {
+    "en": "Started as a site to give you notifications whenever your favourite anime goes live, Anime Notifier is now also useable as an anime list, with an extention so you don't have to leave your anime to keep your list up-to-date."
+  },
+  "url": "notify.moe",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/hkP5B5D.png",
+  "thumbnail": "https://i.imgur.com/f9POmUU.png",
```

Comment:
that's the weirdest thumbnail that I ever seen. Can you change it to site screenshot?